FEATURES.md

```diff
@@ -0,0 +1,97 @@
+
+# Feature proposal process:
+
+The feature proposal is an issue filed on a repo in OpenSearch which outlines the statement of the problem (technical design is not needed).The issue doesn’t have to contain implementation or design details but it's recommended to include ideas of possible fixes. The primary goal is to brainstorm the solutions along with the community beginning from this feature proposal phase. The issue is tagged “[feature-proposal]” and each repo should have their own template or a fallback on a standard feature proposal template. The maintainers will triage the feature proposal and ask follow-up questions as needed. The maintainer’s job is to determine if the proposal meets the end-result criteria:
```

Comment:
A feature proposal ...

in opensearch-project (to not confuse with OpenSearch repo)

Add space after periods.